samples/drivers/can/babbling/boards/imx93_evk_mimx9352_a55.conf

```diff
@@ -0,0 +1,6 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_I2C=y
+CONFIG_MFD=y
+CONFIG_GPIO_HOGS=y
```

Comment:
we don't need to enable all these items, they are enabled by dts by default.

Reply:
so conf is not needed